### PR TITLE
add dirname options

### PR DESCRIPTION
**Pull request by lynhan318**

### CODE REVIEW

File: gen/atlas.tmpl [hunk 1]
```diff
@@ -21,6 +21,7 @@ jobs:
       - uses: ariga/atlas-action@v0
         with:
           dir: '{{ .Path }}'
+          dir-name: '{{ .DirName }}'
```
Comment (a8m) [inline gen/atlas.tmpl:24]:
```suggestion
{{- with .DirName }}
          dir-name: '{{ . }}'
{{- end }}
```

Make it optional in the template.

Comment (a8m) [reply gen/atlas.tmpl:24]: Also, add it in the sync action below.

File: gen/atlas.tmpl [hunk 2]
```diff
@@ -33,5 +36,8 @@ jobs:
       - uses: ariga/atlas-sync-action@v0
         with:
           dir: '{{ .Path }}'
+{{- with .DirName }}
+          dir-name: '{{ . }}'
```
Comment (a8m) [inline gen/atlas.tmpl:40]:
```suggestion
          name: '{{ . }}'
```
In sync action, this is named as 'name'.

Comment (a8m) [reply gen/atlas.tmpl:40]: https://github.com/ariga/atlas-sync-action/blob/master/action.yml#L10

Comment (lynhan318) [reply gen/atlas.tmpl]: Oh didn't notice that!, Thanks @a8m .

**Comment by masseelch** [comment]
Closed in favor of #316.

**Pull request closed without merging**